scripts/proto.sh

```diff
@@ -71,3 +71,4 @@ protoc \
 --grpc-gateway_out=logtostderr=true,$TYPES:. \
 --swagger_out=logtostderr=true,$TYPES,allow_merge=true,simple_operation_ids=true,json_names_for_fields=true,merge_file_name=./pkg/api/binoculars/api:. \
 pkg/api/binoculars/binoculars.proto \
+
```

Comment:
nit: appears to be an unintentional change, but it does no harm

Reply:
I think this might have been auto-added my editor (Vim with several plugins) or IDE (VS Code) - I wonder if it didn't like the final line of the file ending with a `\` char?

Reply:
Dropping the `\` on line 73 would be a better change than this one, looking  more closely. But also :shrug:

Reply:
The better change is in commit 58af24ea